web_src/css/base.css

```diff
@@ -266,7 +266,7 @@ body {
   color: var(--color-text);
   background-color: var(--color-body);
   tab-size: var(--tab-size);
-  overflow-y: auto;
+  overflow: auto;
```

Comment:
Do we actually need to set `overflow` at all?

Reply:
See https://github.com/go-gitea/gitea/pull/25231#issuecomment-1588983812

> Use `overflow: auto` for `body` instead of `overflow-y: auto`. Because `overflow: overlay` has been used for long time, it behaves like `overflow: auto`.


Or do you mean we just remove it?

Reply:
Yeah I mean just remove it as a test. Body should overflow just fine with default `overflow: visible` I guess.

Reply:
But Fomantic UI has `overflow-x` in its `semantic.css` by default .... we still have to override them.

I still think `overflow: auto` is the right thing at the moment.

Without `overflow: auto` (the first screenshot), if anything overflows, the users do not have any chance to see it.

With `overflow: auto` (the second screenshot), at least the users could see the contents (there is a scrollbar)

<details>

<img width="273" alt="image" src="https://github.com/go-gitea/gitea/assets/2114189/9daaffab-e64e-436b-a108-6d7e53fd19b4">

<img width="272" alt="image" src="https://github.com/go-gitea/gitea/assets/2114189/a786adf5-26bf-49ae-9c2a-3d052c053545">

</details>

Reply:
Would overriding to `overflow: visible` work? I just would like to be as close as possible to browser defaults.

Reply:
I think you can add `@pageOverflowX: visible` to `web_src/fomantic/_site/globals/site.variables` and rebuild fomantic to avoid having to override.

Reply:
I wonder what's causing fomantic to include those vendor prefixes, it does not do that when I run it.

Reply:
Actually it may be something on my other machine, it does build with vendor prefixes right now for me.

Reply:
I added a lockfile to the fomantic folder, that seems to have fixed it. My theory is that some dependency of fomantic changed prefixing behaviour recently.